### ISSUE TITLE
10.2 Blueprint POST Nested route does not present id to created object

### ISSUE THREAD

**Issue by MickCrozier** [balderdashy/sails#2114]
Post: /owner/1/pet/
The new pet object does not get {owner: 1}

The association is not made

**Comment by CWyrtzen** [comment]
Does updating to v.11 solve this issue?

**Comment by sailsbot** [comment]
Thanks for posting, @MickCrozier.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
